### DISCUSSION TITLE
使用 JSON Schema 开发表单，如何在表单控件下添加一段纯文本做引导说明？

### DISCUSSION THREAD

**Discussion by roark47** [alibaba/formily#1695] [Q&A 1.x] [answered]
如题，翻遍文档没找到 solution，麻烦回复下，thx。

**Comment by janryWang** [comment, marked as answer]
description

**Reply by roark47** [reply]
![](https://gw.alicdn.com/imgextra/i1/O1CN01CkigzX1t1E6uYk3oA_!!6000000005841-0-tps-648-312.jpg)

description 的文字不应该受值校验结果而改变样式吧？

**Reply by janryWang** [reply]
嗯，升级2.x

**Reply by roark47** [reply]
2.x 是个 break change 版本，我没法直接升级，像这种 1.x 的 bug 还是要 fix 的吧。

**Reply by janryWang** [reply]
> 2.x 是个 break change 版本，我没法直接升级，像这种 1.x 的 bug 还是要 fix 的吧。

这又不是bug，1.x一直以来都是这样的，改掉反而是break change

**Reply by roark47** [reply]
字段校验失败应该是错误提示文字标红而不是描述文字标红，1.x 一直以来都是这样是不正确的，不然 2.x 也没必要改过来。

**Comment by janryWang** [comment]
那你用x-component-props.extra吧，这个不会标红

**Reply by roark47** [reply]
好的，THX

**Reply by darkmice** [reply]
1.x也能用description来配置

![image](https://user-images.githubusercontent.com/1502088/125590025-ac2d9270-509c-40c3-bcd2-84aa20c96d35.png)
